packages/react/src/components/DataTable/TableHeader.js

```diff
@@ -18,24 +18,27 @@ import { sortStates } from './state/sorting';
 const { prefix } = settings;
 
 const translationKeys = {
-  iconDescription: 'carbon.table.header.icon.description',
+  buttonDescription: 'carbon.table.header.icon.description',
```

Comment:
Note: Tried to change 'carbon.table.header.icon.description' to 'carbon.table.header.button.description' since this label is no longer the icon, but the button. However, I got a warning about the public api, so I left it as is.
![Screen Shot 2020-12-01 at 4 34 44 PM](https://user-images.githubusercontent.com/32556167/100804841-296cd580-33f3-11eb-9a4c-259d2644c69f.png)